opamp/observiq/observiq_client.go

```diff
@@ -136,8 +136,12 @@ func (c *Client) Connect(ctx context.Context) error {
 	settings := types.StartSettings{
 		OpAMPServerURL: c.currentConfig.Endpoint,
 		Header: http.Header{
-			"Authorization": []string{fmt.Sprintf("Secret-Key %s", c.currentConfig.GetSecretKey())},
-			"User-Agent":    []string{fmt.Sprintf("observiq-otel-collector/%s", version.Version())},
+			"Authorization":  []string{fmt.Sprintf("Secret-Key %s", c.currentConfig.GetSecretKey())},
+			"User-Agent":     []string{fmt.Sprintf("observiq-otel-collector/%s", version.Version())},
+			"OpAMP-Version":  []string{opamp.Version()},
+			"AgentID":        []string{c.ident.agentID},
```

Comment:
I think I would use `Agent-ID` to be consistent with the other `Agent-` headers. Otherwise I think this looks great.

Reply:
@andykellr Updated.